src/functionalTest/java/uk/gov/hmcts/reform/laubackend/idam/serenityfunctionaltests/steps/BaseSteps.java

```diff
@@ -154,5 +154,22 @@ public String thenBadResponseIsReturned(Response response, int expectedStatusCod
         return TestConstants.SUCCESS;
     }
 
+    @Step("When the GET service is invoked")
+    public Response whenTheGetServiceIsInvoked(
+        String endpoint,
+        String serviceToken,
+        String authorizationToken,
+        Map<String, String> queryParams
+    ) throws JsonProcessingException {
```

Comment:
The method does not throw JsonProcessingException.

Reply:
Done